hw11/css/styles_part2.css

```diff
@@ -0,0 +1,144 @@
+* {
+  padding: 0;
+  margin: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: ProximaNovaRegular;
```

Comment:
Would be nice to have fallback fonts to use in case ProximaNovsRegular are missing
https://css-tricks.com/css-basics-fallback-font-stacks-robust-web-typography/

Reply:
Fixed, haven't added it since fonts are part of the project and were declared in the file, but agree, it would be nice to add fallback in case they are missed.